src/components/Header/Header.stories.tsx

```diff
@@ -8,6 +8,7 @@ import { LinkVariants } from '../Link/utils';
 import { px } from '../../utils';
 import UserManagement from '../UserManagement/UserManagement';
 import Search from '../Search/Search';
+import { AuthState } from '../UserManagement/types';
```

Comment:
Should we prefer to use `import type ...` when importing only types? It helps to avoid some potential issues with importing types, but probably more importantly, `import type` is guaranteed to get removed from any bundled code, whereas regular `import` may not

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

Reply:
since this is both a type and a constant I can't do the type import.

Reply:
Ah, my mistake. I didn't see that it was being used for both... Are we already using `import type` in other places?